bundles-lib/pom.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
All of `bundles-lib` came from Nifi's NARs right?  Can we add a README and document that fact?  Or maybe you already have it doc'd elsewhere?